UI-extra/index.html

```diff
@@ -200,29 +213,29 @@ <h1 class>LENOVO FLEX, Intel Core i3 14" FHD Multi Point Touch</h1>
                         <div class="padding-top-m padding-bottom-m">
                             <div class="flex-box padding-all-m">
```

Comment:
Add `flex-box__start` so the items lines up at the top next to each other. For example, first item has only 1 line of text and 2nd item has 2 lines of text wrap. Right now, they are vertically centered to each other